nexus-sdk/build.gradle

```diff
@@ -2,6 +2,7 @@
 plugins {
     id 'java-library'
     id 'com.diffplug.spotless'
+    id 'com.vanniktech.maven.publish' version '0.29.0'
```

Comment:
There is not good Gradle support for publishing on Central yet: https://central.sonatype.org/publish/publish-portal-gradle/. So https://github.com/vanniktech/gradle-maven-publish-plugin/ is the best of the available options based on trying multiple out.